docs/debug/README.md

```diff
@@ -40,7 +40,7 @@ spec:
                   fieldPath: metadata.namespace
             - name: SECRET_NAME
               value: kruise-webhook-server-secret
-          image: $DOCKERID/kruise:test
+          image: openkruise/kruise:test
```

Comment:
You should not upload your test image into openkruise category. Try to create your personal `DOCKERID` and put the test image into it.